src/bci_build/package/bind.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+
+from bci_build.os_version import ALL_NONBASE_OS_VERSIONS
+from bci_build.os_version import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import DOCKERFILE_RUN
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+
+_BIND_FILES = {
+    "entrypoint.sh": (
+        (_bind_dir := Path(__file__).parent / "bind") / "entrypoint.sh"
+    ).read_text(),
+    "healthcheck.sh": (_bind_dir / "healthcheck.sh").read_text(),
+}
+
+BIND_CONTAINERS = [
+    ApplicationStackContainer(
+        name="bind",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version=9,
+        pretty_name="ISC BIND 9",
+        version_in_uid=False,
+        package_list=["bind"],
+        additional_versions=[
+            (_bind_minor_re := "%%bind_major_minor%%"),
+            (_bind_patch_re := "%%bind_major_minor_patch%%"),
+        ],
+        replacements_via_service=[
+            Replacement(_bind_minor_re, "bind", parse_version=ParseVersion.MINOR),
+            Replacement(_bind_patch_re, "bind", parse_version=ParseVersion.PATCH),
+        ],
+        extra_files=_BIND_FILES,
+        env={
+            # copy-pasta from /etc/sysconfig/named
+            "RNDC_KEYSIZE": 512,
+            "NAMED_ARGS": "",
+            "NAMED_INITIALIZE_SCRIPTS": "",
+            # need to set this one so that we can override it
+            "NAMED_CONF": "/etc/named.conf",
+        },
+        custom_end=rf"""COPY entrypoint.sh {(_entrypoint := "/usr/local/bin/entrypoint.sh")}
+COPY healthcheck.sh {(_healthcheck := "/usr/local/bin/healthcheck.sh")}
+{DOCKERFILE_RUN} \
+    chmod +x {_entrypoint}; \
+    chmod +x {_healthcheck};
+
+EXPOSE 53/udp 53/tcp 953/tcp 853/tcp 443/tcp
```

Comment:
didn't we have some exposes_port setting through the template generator ? would prefer that over having it hidden in custom_end as it is less easy to grep.

Reply:
Totally forgot that we had this :facepalm: